src/doc/gui_manual/Preferences/File_Locations.rst

```diff
@@ -0,0 +1,297 @@
+  ANL, ORNL, etc. Often sites will *post* VisIt_ host profile files in places for
+  users to easily find and *install* them. Installing them is just a matter of
+  copying them to ``VUSER_HOME``. In addition, updated profiles can be downloaded
+  and installed automatically by VisIt_ from the :ref:`Host Profiles <host_profiles>`
+  window.
+* Written: When user :ref:`saves settings <How to Save Settings>` or when user
+  hits the **Export Host** button from the :ref:`Host Profile <host_profiles>` window.
+* Read: On VisIt_ startup. All host profiles in ``VUSER_HOME/hosts/host*.xml`` are read
+  on VisIt_ startup but this can be overridden by ``-noconfig``. Users should be
+  aware of this behavior. If the user passes ``-noconfig`` for the purposes of
+  avoiding the loading of preferences, s/he will also be without any host profiles.
+* Format: ASCII `XML <https://en.wikipedia.org/wiki/XML>`_
+
+.. _file_locations_visitrc:
+
+VisIt Run Commands (rc) File
+""""""""""""""""""""""""""""
+* Location and file name: ``VUSER_HOME/visitrc``
+* Purpose: Holds Python code to be executed each time VisIt_ is launched.
+* Written: Whenever user hits the **Update Macros** button in the
+  :ref:`Command Window <Command_Line_Window>`.
+* Read: On VisIt_ startup of the CLI.
+* Format: Python source code. However, there is no ``.py`` file extension in the
+  file name.
+
+Command Window Tabs Script Files
+""""""""""""""""""""""""""""""""
+* Location and file name(s): ``VUSER_HOME/script<K>.py`` where ``K`` is an
+  integer in the range [1...8].
+* Purpose: Hold the python code associated with each tab in the
+  :ref:`Command Window <Command_Line_Window>`.
+* Written: When user :ref:`saves settings <How to Save Settings>`.
+* Read: On VisIt_ startup but this can be overridden by the ``-noconfig``
+* Format: Python source code.
+
+Color Table Files
+"""""""""""""""""
+* Location and file name(s): ``VUSER_HOME/<color-table-name>.ct``
+* Purpose: Store a single color table for easy sharing with other users.
+* Written when the user hits the **Export** button in the
+  :ref:`color table window <fig-MakingItPretty-ColorTables>` from
+  :menuselection:`Controls -> Color table...`.
+* Read: On VisIt_ startup. All color table files in ``VUSER_HOME/*.ct``
+  are read and loaded into VisIt_. However, this behavior is overridden
+  by ``-noconfig``.
+* Format: ASCII `XML <https://en.wikipedia.org/wiki/XML>`_ specifying the
+  :ref:`colors and color control points <Color_tables>` for the color table.
+
+Custom Plugins
+""""""""""""""
+* Location and file name(s): There are separate directories in ``VUSER_HOME``
+  for operator, database and plot plugins. These are
+
+  * ``VUSER_HOME/<visit-version>/<visit-arch>/plugins/operators/``
+  * ``VUSER_HOME/<visit-version>/<visit-arch>/plugins/databases/``
+  * ``VUSER_HOME/<visit-version>/<visit-arch>/plugins/plots/``
+
+  where ``<visit-version>`` and  ``<visit-arch>`` are the VisIt_ version number
```

Comment:
On Windows, the location for 'private' plugins as described here, which is the default mode, is: VUSER_HOME/[operators,databases,plots].

There is the option for 'public' plugins (using the -public option to xml2cmake).
That option will install to VisIt's install location (root install dir/[operators,databases,plots] on Windows).

Reply:
Ok, so are you saying on Windows, there is no version sub-dir and no architecture subdir?

Also, does `-public` also work on Unix/OSX?

Reply:
I don't know if the -public version has been tested in awhile. I do know that xml2cmake still looks for the -public option.  I haven't tested -public on Windows. Probably wouldn't work in an all-user install as users generally don't have write access to the install location in that case.

Reply:
> Ok, so are you saying on Windows, there is no version sub-dir and no architecture sub-dir?

Can I get an answer to this?

Reply:
Definitely no architecture subdir. I am trying to build an external public and will let you know about the version. Sorry for the delay.

Reply:
private plugins saved to VUSER_HOME/[operators,databases,plots]
no arch, no version

Reply:
Ok, I wrote assuming that so I think its in better shape now.